boards/seeeduino_arch-pro/Makefile.include

```diff
@@ -11,6 +11,7 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 DEBUG_ADAPTER ?= dap
 
 # this board uses openocd
+export IMAGE_FILE = $(HEXFILE)
```

Comment:
Looks unrelated. Maybe #8845 will help ?

Reply:
It's needed to flash the HEX file instead of the ELF file. #8845 would help, yes.

Reply:
I would prefer to make this change in a separate PR. Is it ok for you ?

Reply:
I'll put this in another PR.

Reply:
Created #8866.